src/components/CartTableRow/CartTableRow.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {
+  TableData,
+  ProductDetailsWrapper,
+  ProductImageWrapper,
+  Image,
+  ProductDetailsTextWrapper,
+  QuantityWrapper,
+  RemoveButton,
+} from './CartTableRow.styles';
+
+export const CartTableRow = ({item, quantity}) => {
+  const {name, price, images, id} = item;
+  const totalPrice = (price * quantity).toFixed(2);
+
+  return (
+    <tr>
+      <TableData>
+
+        <ProductDetailsWrapper>
+          <ProductImageWrapper>
+            <Image src={images[0]} alt={'Picture of the product' + {name}} />
+          </ProductImageWrapper>
+          <ProductDetailsTextWrapper>
+            <p>{name}</p>
+            <p>Product code: {id}</p>
+          </ProductDetailsTextWrapper>
+        </ProductDetailsWrapper>
+      </TableData>
+
+      <TableData>
+        <QuantityWrapper>
+          <input
+            defaultValue={quantity}
+            type='number'
+            min={0}
+          />
```

Comment:
I cannot validate this functionality because the input does not do what is required

> The "Quantity" input only allows positive numbers and affects the total cost of the product, the total items, and the total cost in the "Summary" section. (10 points)

What was expected from this point is: calculate the total cost by modifying the number of elements.